t/pg.t

```diff
@@ -295,6 +295,15 @@ is $job->id, $id, 'right id';
 ok $job->fail,   'job failed';
 ok $job->remove, 'job has been removed';
 is $job->info,   undef, 'no information';
+$id = $minion->enqueue(add => [6, 9]);
+$job = $worker->dequeue(0);
+like $job->info->{delayed}, qr/^[\d.]+$/, 'has delayed timestamp';
+ok $job->fail, 'job failed';
+ok $job->retry({delay => 10}), 'job retried with delay';
+is $job->info->{retries}, 1, 'job has been retried once';
+ok $job->info->{delayed} > time + 9, 'job delayed';
```

Comment:
This test does not leave a lot of room for time differences between servers, i think all other delay tests leave at least 100 seconds.